src/content/docs/fr/concepts/islands.mdx

```diff
@@ -1,12 +1,13 @@
 ---
-title: Les îles Astro
-description: "Les îles Astro (alias Component Islands) sont un modèle d'architecture web dont Astro est le pionnier. “L'architecture Islands“ a été inventée par l'architecte frontend d'Etsy, Katie Sylor-Miller, en 2019, et développée par le créateur de Preact, Jason Miller."
+title: Architecture des îles
+description: Découvrez comment l'architecture en îlots d'Astro contribue à la rapidité des sites.
 i18nReady: true
 ---
 
 import IslandsDiagram from '~/components/IslandsDiagram.astro';
+import ReadMore from '~/components/ReadMore.astro';
 
-Astro a mis au point et popularisé une architecture frontale appelée **Islands**. L'architecture Islands permet d'améliorer les performances frontales en évitant les modèles JavaScript monolithiques et en supprimant automatiquement tout JavaScript non essentiel de la page. Les développeurs continuent d'utiliser leurs composants d'interface utilisateur et frameworks préférés avec Astro et bénéficient toujours de ces avantages.
+Astro a contribué à la création et à la popularisation d'un nouveau modèle d'architecture Frontend appelé **Islands Architecture**. L'architecture Islands fonctionne en rendant la majorité de votre page en HTML rapide et statique, avec de plus petits « îlots » de JavaScript ajoutés lorsque l'interactivité ou la personnalisation est nécessaire sur la page (un carrousel d'images, par exemple). Cela permet d'éviter les charges utiles JavaScript monolithiques qui ralentissent la réactivité de nombreux autres frameworks Web JavaScript modernes.
```

Comment:
nit: Since we translate `Islands` everywhere, maybe we should here to:
```suggestion
Astro a contribué à la création et à la popularisation d'un nouveau modèle d'architecture Frontend appelé **Islands Architecture**. L'architecture en îles fonctionne en rendant la majorité de votre page en HTML rapide et statique, avec de plus petits « îlots » de JavaScript ajoutés lorsque l'interactivité ou la personnalisation est nécessaire sur la page (un carrousel d'images, par exemple). Cela permet d'éviter les charges utiles JavaScript monolithiques qui ralentissent la réactivité de nombreux autres frameworks Web JavaScript modernes.
```
The first occurrence (`Islands Architecture`) is fine, it'll help with the search!